skyportal/utils/calculations.py

```diff
@@ -269,3 +294,67 @@ def radec2lb(ra, dec):
     galactic_l = np.arctan2(y, x)
     galactic_b = np.arctan2(z, (x * x + y * y) ** 0.5)
     return np.rad2deg(galactic_l), np.rad2deg(galactic_b)
+
+
+def hms_to_deg(input):
+    """
+    Convert hours, minutes, seconds to decimal degrees.
+
+    Parameters
+    ----------
+    h : float
+        Hours
+    m : float
+        Minutes
+    s : float
+        Seconds
+
+    Returns
+    -------
+    float
+        Decimal degrees
+    """
+    h, m, s = input.split(" ")
+    h, m, s = float(h), float(m), float(s)
+    return 15 * (h + m / 60 + s / 3600)  # Multiply by 15 to convert hours to degrees
+
+
+def dms_to_deg(input):
+    """
+    Convert degrees, arcminutes, arcseconds to decimal degrees.
+
+    Parameters
+    ----------
+    d : float
+        Degrees
+    m : float
+        Arcminutes
+    s : float
+        Arcseconds
+
+    Returns
+    -------
+    float
+        Decimal degrees
+    """
+    d, m, s = input.split(" ")
+    d, m, s = float(d), float(m), float(s)
+    sign = 1 if d >= 0 else -1
+    return sign * (abs(d) + m / 60 + s / 3600)
+
+
+def radec_to_healpix(row: dict | pd.Series) -> int:
```

Comment:
Any need to also vectorize this?

Reply:
you can only vectorize (at leat with np.vectorize) pure numpy/python functions, won't work here at least not as expected I bet